### PR TITLE
Not Found & Server Error pages

### PR DESCRIPTION
**Pull request by hashemt**
Relates #98

### CODE REVIEW

File: client/src/components/Pages/NotFound/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import pageNotFound from '../../../assets/404.png';
+import './style.css';
+
+const NotFound = () => {
+  return (
+    <section>
+      <img src={pageNotFound} alt="Not Found 404" className="notFound__img" />
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/NotFound/index.js]:
```suggestion
      <img src={pageNotFound} alt="Page Not Found 404" className="notFound__img" />
```

File: client/src/components/Pages/NotFound/style.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    color: #FFF;
+    font-size: 20px;
+    padding: 3px;
+    border-radius: 10px;
+    display: block;
+    width: 50%;
+    margin: 25px auto;
+    text-align: center;
+    cursor: pointer;
+}
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/NotFound/style.css]: add new line

File: client/src/components/Pages/ServerError/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import error500 from '../../../assets/error500.png';
+import './style.css';
+
+const ServerError = () => {
+  return (
+    <section>
+      <img src={error500} alt="Server Error 500" className="serverError__img" />
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/ServerError/index.js]:
```suggestion
      <img src={error500} alt="Internal Server Error 500" className="serverError__img" />
```

File: client/src/components/Pages/NotFound/style.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
+.notFound__img {
+    width:100%;
```
Comment (mossa-Sammer) [inline client/src/components/Pages/NotFound/style.css]:
```suggestion
    width: 100%;
```

File: client/src/components/Pages/NotFound/style.css [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    margin-top:30%;
```
Comment (mossa-Sammer) [inline client/src/components/Pages/NotFound/style.css]:
```suggestion
    margin-top: 30%;
```

File: client/src/components/Pages/NotFound/style.css (hunk 4 above)
Comment (mossa-Sammer) [inline client/src/components/Pages/NotFound/style.css]:
```suggestion
    width: 100%;
```

File: client/src/components/Pages/ServerError/style.css [hunk 6]
```diff
@@ -0,0 +1,17 @@
+.serverError__img {
+    width:100%;
```
Comment (mossa-Sammer) [inline client/src/components/Pages/ServerError/style.css]:
```suggestion
    width: 100%;
```

File: client/src/components/Pages/ServerError/style.css [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    margin-top:30%;
```
Comment (mossa-Sammer) [inline client/src/components/Pages/ServerError/style.css]:
```suggestion
    margin-top: 30%;
```

**Pull request merged**